assets/frontend/users/services.js

```diff
@@ -46,6 +47,13 @@
       return $http.get(API_URL + 'logout/')
     };
 
+    function userSignUp (data) {
+      return $http.post(API_URL + 'signup/', data)
+      .then(function (resp) {
+        $state.go('login');
```

Comment:
don't put the state in here. This action should happen on the controller.

controller = user triggered interaction
service = app logic